tools/sklearn/test-data/pca_kernel_output2.dat

```diff
@@ -0,0 +1,300 @@
+-789.0683 244.4933 86.3224 14.2457 336.2516 -48.3858 104.9891 115.2530
+-721.5195 53.8094 164.5249 -213.6399 283.9916 -180.7496 -85.6662 -56.4151
+-731.7468 192.1683 -263.2790 -109.8660 -131.7332 -389.4955 60.0203 53.8620
+-789.8859 246.9337 102.2221 -46.5027 312.8768 -90.5573 -21.4048 101.4942
+-818.3874 350.2119 -440.5530 165.6495 224.8707 93.6159 -25.2826 7.4475
+-833.1559 459.6316 -1178.7885 276.9423 -47.5942 118.5477 -287.9134 -43.3138
```

Comment:
is this really a tabular file?

Reply:
@lorrainealisha75, could you also look at this? The output files should be tab-separated.